### DISCUSSION TITLE
posts/naverlogin/

### DISCUSSION THREAD

**Discussion by giscus** [22000510/22000510.github.io#8] [General]
# posts/naverlogin/

selenium 이용해 네이버 자동로그인 해보기 파이썬 selenium 코드를 이용해서 네이버 자동로그인 하는 방법을 알아보도록 하겠습니다. 어렵지 않은 코드지만 네이버 자동 로그인과 자동입력 방지 문자 안나오도록 하는 법에 대해 포스팅 해보겠습니다.

https://22000510.github.io/posts/naverlogin/

**Comment by 22000510** [comment]
연습 채팅 입니다.
